extensions/pkg/controller/healthcheck/general/managed_resource.go

```diff
@@ -49,11 +48,6 @@ func (healthChecker *ManagedResourceHealthChecker) InjectSeedClient(seedClient c
 	healthChecker.seedClient = seedClient
 }
 
-// InjectShootClient injects the shoot client
-func (healthChecker *ManagedResourceHealthChecker) InjectShootClient(shootClient client.Client) {
```

Comment:
nice, so for managed resource health checks, a shoot client is not created any more?

Reply:
Yes, a health checker now needs to have a `InjectShootClient` function to get the client.